README.md

```diff
@@ -268,7 +268,11 @@ This object defines templates (functions) that are used for displaying parts of
 
 `inputValue` is a function that receives one argument, the currently selected suggestion. It returns the string value to be inserted into the input.
 
-`suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML. :warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
+`suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML.
+
+:warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
```

Comment:
What does 'arbitrary' mean here? Random?

I get that it's the HTML that needs to be trusted and accessible, but that might not be totally clear at first glance.

Reply:
This line also isn't new, just split up.

It's a bit wordier, but what about…

>  To prevent [cross-site scripting vulnerabilities](https://en.wikipedia.org/wiki/Cross-site_scripting) ensure that the string returned by this function contains only trusted or sanitised input. You should also ensure that any additional HTML markup is accessible.

Reply:
Yeah, that sounds good. Maybe just the lightest of tweaks:

> To prevent [cross-site scripting vulnerabilities](https://en.wikipedia.org/wiki/Cross-site_scripting), make sure the string returned by this function contains only trusted or sanitised input. You should also make sure any additional HTML markup is accessible.

(Sorry if this seems finicky - we're encouraged to write 'make sure' (plainer alternative) instead of 'ensure', and remove redundant _that_s.)